assignment1/Assignment1.ipynb

```diff
@@ -1111,7 +1111,16 @@
       "source": [
         "# Please note that this code needs only to be run in a fresh runtime.\n",
         "# However, it can be rerun afterwards too.\n",
+        "# Here we download and install localy two libraries:\n",
+        "# -  gdown is library similar to curl and wget but allow to download big files from google drive\n",
+        "#    two previous apps have problem with this because of google warning that file is too big to scan it with antyvirus.\n",
+        "# -  httpimport allow to import modules from web using HTTP/HTTPS.\n",
```

Comment:
```suggestion
        "# Here we download and install locally two libraries:\n",
        "# -  gdown is a library similar to curl and wget but it allows to download big files from google drive\n",
        "#    two previous apps have problem with this because of google warning that file is too big to scan it with antivirus.\n",
        "# -  httpimport allows to import modules from web using HTTP/HTTPS.\n",
```